src/infinity/libraries/LibRetire.sol

```diff
@@ -285,6 +287,14 @@ library LibRetire {
                 amount,
                 details
             );
+        } else if (LibCMARKCarbon.isValid(creditToken)) {
+            // Retire the carbon
+            LibCMARKCarbon.retireCMARK(
+                address(0), // Direct retirement, no pool token
```

Comment:
is there a reason to keep the address(0) for the poolToken param? Maybe there is for clarity(?). If it will always be irrelevant for cmark credits we can just take that param out of that function altogether.

Reply:
ah yeah good catch I forgot to remove this here